src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,32 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
+def merge(arrA, arrB):
+    elements = len(arrA) + len(arrB)
     merged_arr = [0] * elements
-    # TO-DO
-    
+    combined = arrA + arrB
+
+    for index in range(0, len(merged_arr)):
+        merged_arr[index] = combined[index]
+
+    merged_arr.sort()
+
     return merged_arr
 
+merge(
+    [1, 5, 8, 4, 2, 9, 6, 0, 3, 7],
+    [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+)
 
 # TO-DO: implement the Merge Sort function below USING RECURSION
-def merge_sort( arr ):
-    # TO-DO
 
-    return arr
+
+def merge_sort(arr):
+    
+    if (len(arr) <= 1):
+        return arr
+
```

Comment:
Need to split arr into a left and right.
Recursively call sorted_left = merge_sort(left) & sorted_right = merge_sort(right)
Once you have those sorted portions of the original input, arr, pass into the merge function and return the output
return merge(sorted_left, sorted_right)